clang/lib/CodeGen/CodeGenFunction.h

```diff
@@ -5149,6 +5152,9 @@ class CodeGenFunction : public CodeGenTypeCache {
                            SourceLocation ArgLoc, AbstractCallee AC,
                            unsigned ParmNum);
 
+  /// EmitWriteback - Emit callbacks for function.
+  void EmitWritebacks(CodeGenFunction &CGF, const CallArgList &Args);
```

Comment:
This isn't a static method, but it's signature and use make it look like you meant for it to be.  It seems like it could just be a regular method though.

```suggestion
  void EmitWritebacks(const CallArgList &Args);
```